scripts/build/Platform/Windows/pipeline.json

```diff
@@ -4,7 +4,9 @@
         "LY_3RDPARTY_PATH": "C:/ly/3rdParty",
         "TIMEOUT": 30,
         "WORKSPACE": "D:/workspace",
-        "MOUNT_VOLUME": true
+        "MOUNT_VOLUME": true,
+        "AWSI_DEPLOY_REGION": "us-west-2",
```

Comment:
The parameters defined in pipeline.json won't show up as a Jenkins build parameters, so we still have the same problem with putting them in build_config.json.

To make these parameter be Jenkins build parameters, they need to be defined in Jenkinsfile, but by doing this, all AR jobs will have these Jenkins build parameters because all AR jobs are using the same Jenkinsfile. I'm not against this approach but it might cause some confusion to people who manually triggers an AR build.

Reply:
Synced offline. We will create a separate jenkinsfile pipeline instead.